### PR TITLE
added the map

### PR DESCRIPTION
**Pull request by Thunderducky**
I added a new map please review!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,9 +21,100 @@
...
+                </div>
+            </div>
+        </div>
     <div>
-        <img class="img img-thumbnail" src="https://i.imgur.com/wWDjJEB.jpg" alt="">
+        
     </div>
+    <script type="text/javascript">
+        setTimeout(() => {
+            alert("Actually it's squirtle");
```
Comment (baggiodev) [inline index.html]: You gotta meet me halfway

**Review by baggiodev — COMMENTED** [review]
Please fix the alert.

File: index.html (hunk 1 above)
Comment (Thunderducky) [reply index.html]: Okay, I'll fix it

**Review by baggiodev — APPROVED** [review]
Great job looks good!

**Pull request merged**